### PR TITLE
[17.0][MIG] intrastat_product

### PR DESCRIPTION
**Pull request by dreispt**
Supersedes #261

### CODE REVIEW

File: intrastat_product/models/intrastat_transaction.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    code = fields.Char(required=True)
+    description = fields.Text(translate=True)
+    # intrastat.transaction are shared among companies by default
+    company_id = fields.Many2one("res.company")
+    active = fields.Boolean(default=True)
+
+    @api.depends("code", "description")
+    def _compute_display_name(self):
+        for this in self:
+            name = this.code
```
Comment (GuillemCForgeFlow) [inline intrastat_product/models/intrastat_transaction.py]:
```suggestion
            name = this.code or ""
```
Following with comment here https://github.com/OCA/intrastat-extrastat/pull/261#discussion_r1540616137. I guess you can add it here.

**Review by GuillemCForgeFlow — CHANGES_REQUESTED** [review]
After this is changed, the rest looks good to be merged IHMO 👍🏿

**Comment by etobella** [comment]
@dreispt I think with a small typo change we can get this merged :wink:

**Comment by GuillemCForgeFlow** [comment]
> @dreispt I think with a small typo change we can get this merged 😉

Well it's not actually a typo, if the change is not added it will fail upon the creation of a new Transaction Type, therefore we need the change first.

image

**Comment by etobella** [comment]
Well, let's say it is small change :wink:

**Comment by juanmagil** [comment]
When will this migration to the 17.0 branch be merged?

File: intrastat_product/models/intrastat_transaction.py (hunk 1 above)
Comment (dreispt) [reply intrastat_product/models/intrastat_transaction.py]: This bust be a pre-existing bug affecting previous versions too.
I added a fix commit.

**Comment by dreispt** [comment]
Lint fixes were added and build is green now.
I believe it can be merged based on the approvals at #261

**Review by gurneyalex — APPROVED** [review]
ok for me.

**Pull request merged**